draft-enghardt-panrg-path-properties.md

```diff
@@ -66,63 +64,62 @@ in Path-Aware Networking {{I-D.irtf-panrg-questions}}, which is a product of the
+Domain path properties relate to path elements within the same Autonomous System (AS), thus, in the same administrative domain as an endpoint considering them.
+
+Due to the potential physical proximity and pre-existing trust or contractual relationships between endpoints and path elements within the same administrative domain, domain properties may be more easily available to the endpoint than the properties of path elements outside of its administrative domain.
 
-A large amount of information about domain properties exists.
-Properties related to configuration can be queried using provisioning domains (PvDs).
-A PvD is a consistent set of network configuration information as defined in {{RFC7556}}, e.g., relating to a local network interface.
-This may include source IP address prefixes, IP addresses of DNS servers, name of an HTTP proxy server, DNS suffixes associated with the network, or default gateway IP address.
-As one PvD is not restricted to one local network interface, a PvD may also apply to multiple paths.
+Furthermore, endpoints may be able to influence both which domain they are in and which path elements in this domain to connect to, and they may be able to influence the properties of path elements within this domain.
+For example, a user might select between multiple potential path elements which are directly adjacent by selecting between multiple available WiFi Access Points. When connected to an Access Point, they may move closer to a it to enable their device to use a different access technology, potentially increasing the data rate available to the device.
+Another example is a user changing their data plan to reduce the Monetary Cost to transmit a given amount of data across a network.
 
-Access Technology present on the path:
-: The lower layer technology on the first hop, for example, WiFi, Wired Ethernet, or Cellular. This can also be more detailed, e.g., further specifying the Cellular as 2G, 3G, 4G, or 5G technology, or the WiFi as 802.11a, b, g, n, or ac. These are just examples, this list is not exhaustive, and there is no common index of identifiers here. Note that access technologies further along the path may also be relevant, e.g., a cellular backbone is not only the first hop, and there may be a DSL line behind the WiFi.
+
+Access Technology:
+: The physical or link layer technology used on one or multiple path elements for transmitting a flow. The Access Technology may be given in an abstract way, e.g., as a WiFi, Wired Ethernet, or Cellular link, or as a specific technology, e.g., as a 2G, 3G, 4G, or 5G cellular link, or an 802.11a, b, g, n, or ac WiFi link. Other path elements relevant to the access technology may include on-path devices, such as elements of a cellular backbone network. Note that there is no common registry of possible values for this property.
```

Comment:
I think for an endpoint this only makes sense within the same domain, as the endpoint might be able to influence that technology. In the backbone, the endpoint doesn't care if some link is 1G or 10G Ethernet, if there's Fiber involved or not, etc., does it? At most it cares about congestion on this link and the resulting latency or data rate bottleneck. 

Then I guess I'll change first sentence of the definition to only include path elements in the same domain. I don't think that we need to add a further definition to this section, but perhaps we need a definition for "domain" in our Terminology section.

Reply:
That's true.

I'm not sure if our definition of domain in the Terminology section is not confusing since we mention both administrative control and administrative domain which do not relate to the same concept.